frontend/src/Settings/MediaManagement/MediaManagement.js

```diff
@@ -130,6 +130,17 @@ class MediaManagement extends Component {
                         isAdvanced={true}
                         size={sizes.MEDIUM}
                       >
+                        <FormLabel>Use Reflinks instead of Copy</FormLabel>
```

Comment:
I think we should at least use the conditional `isMono`, even if it's not perfect

Reply:
I've added it for now. This is to be removed when the Windows devs on Lidarr add support for reflink in ReFS. I'm not adding that myself: can't test, too much code, no interest :-)

Reply:
Seems to be something wrong with 398cfe0824, looking at it now :-)